pkg/webhook/validators.go

```diff
@@ -0,0 +1,67 @@
+package webhook
```

Comment:
Let's move the validating webhook code to its own package `pkg/validator`. `pkg/webhook` is meant to house common webhook utility routines between the validating and mutating webhook configurations.